Justfile

```diff
@@ -33,7 +33,7 @@ kill-server:
 test-e2e: && kill-server
   trunk serve --port=5274 & pid=$!; echo "$pid" > .pid
   sleep 5
-  CI=true npx cypress run
+  CI=true npx cypress run --env failOnSnapshotDiff=false
```

Comment:
Cypress is not respecting the viewport in CI. It's a known issue: https://docs.cypress.io/api/plugins/browser-launch-api#Set-screen-size-when-running-headless. For what we're trying to do (automate visual testing in mobile / desktop) we will have to disabled failures in CI for now. It is still valuable to have automated comparison, even if that is just local for now.